sys-devel/clang/clang-12.0.1.ebuild

```diff
@@ -130,7 +130,7 @@ check_distribution_components() {
 
 				all_targets+=( "${l}" )
 			fi
-		done < <(ninja -t targets all)
+		done < <("${NINJA:-ninja}" -t targets all)
```

Comment:
Why do you need the fallback value?

Reply:
I was trying to be overly cautious, but yea its pointless and I removed the fallback.